### ISSUE TITLE
Something about baseline

### ISSUE THREAD

**Issue by ToxicDoubleH** [HeliosZhao/M3L#6]
Hi, nice job!
I am curious about how to implement the baseline.
`
for dataset in datasets:
    for index in range(iters):
        loss = cls + tri + cent
`
Is the baseline written like this?
And in your Table 4, line 2, as follows
![Y4}4JR6R$@CCBGFHK(ZVCUR](https://user-images.githubusercontent.com/37261572/147423156-6746997c-156c-4f79-bcbd-81f602064cc0.png)
Did you use unify the classification loss of the label space of all datasets ,triplet ans centerloss?
Looking forward to your reply.
